backends/qualcomm/qnn_quantizer.py

```diff
@@ -733,9 +837,199 @@ def _annotate_upsample2d(self, gm: torch.fx.GraphModule) -> None:
+                    else:
+                        input_qspec_map[input_node] = SharedQuantizationSpec(input_node)
+
+                    node.meta[QUANT_ANNOTATION_KEY] = QuantizationAnnotation(
+                        input_qspec_map=input_qspec_map,
+                        output_qspec=SharedQuantizationSpec((input_node, node)),
+                        _annotated=True,
+                    )
+
+    def _annotate_view(self, gm: torch.fx.GraphModule) -> None:
+        op_sources = ["view"]
+        qconfig = self._get_quant_config(op_sources)
+        view_partitions = get_source_partitions(gm.graph, op_sources)
+
+        view_partitions = list(itertools.chain(*view_partitions.values()))
+        for view_partition in view_partitions:
+            view_node = view_partition.output_nodes[0]
+            input_node = view_node.args[0]
+
+            input_qspec_map = {}
+            input_qspec_map[input_node] = qconfig.input_activation
+
+            view_node.meta[QUANT_ANNOTATION_KEY] = QuantizationAnnotation(
+                input_qspec_map=input_qspec_map,
+                output_qspec=SharedQuantizationSpec((input_node, view_node)),
+                _annotated=True,
+            )
+
+    def _annotate_permute(self, gm: torch.fx.GraphModule) -> None:
+        op_sources = ["permute"]
+        qconfig = self._get_quant_config(op_sources)
+        permute_partitions = get_source_partitions(gm.graph, op_sources)
+
+        permute_partitions = list(itertools.chain(*permute_partitions.values()))
+        for permute_partition in permute_partitions:
+            permute_node = permute_partition.output_nodes[0]
+            input_node = permute_node.args[0]
+
+            input_qspec_map = {}
+            input_qspec_map[input_node] = qconfig.input_activation
+
+            permute_node.meta[QUANT_ANNOTATION_KEY] = QuantizationAnnotation(
+                input_qspec_map=input_qspec_map,
+                output_qspec=SharedQuantizationSpec((input_node, permute_node)),
+                _annotated=True,
+            )
+
+    def _annotate_transpose(self, gm: torch.fx.GraphModule) -> None:
+        op_sources = ["transpose"]
+        qconfig = self._get_quant_config(op_sources)
+        transpose_partitions = get_source_partitions(gm.graph, op_sources)
+
+        transpose_partitions = list(itertools.chain(*transpose_partitions.values()))
+        for transpose_partition in transpose_partitions:
+            transpose_node = transpose_partition.output_nodes[0]
+            input_node = transpose_node.args[0]
+
+            input_qspec_map = {}
+            input_qspec_map[input_node] = qconfig.input_activation
+
+            transpose_node.meta[QUANT_ANNOTATION_KEY] = QuantizationAnnotation(
+                input_qspec_map=input_qspec_map,
+                output_qspec=SharedQuantizationSpec((input_node, transpose_node)),
+                _annotated=True,
+            )
```

Comment:
also these can also use propagation

Reply:
We will do it in another pr as aforementioned, thanks